storybook-utilities/markdownTableBuilder.js

```diff
@@ -0,0 +1,32 @@
+const createTable = (tableData) => {
```

Comment:
It might be better to call this something more specific, like markdownDescriptionTableBuilder.